cli/search/search.go

```diff
@@ -90,3 +93,106 @@ func SearchVersions(cmdCtx *cmdcontext.CmdCtx, program string) error {
 	os.Stdout.Write([]byte("master\n"))
 	return err
 }
+
+func getDirFiles(path string) ([]fs.FileInfo, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+	files, err := f.Readdir(0)
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+	return files, err
+}
+
+// RunCommandAndGetOutputDir returns output of command.
+func RunCommandAndGetOutputDir(program string, dir string, args ...string) (string, error) {
+	cmd := exec.Command(program, args...)
+	cmd.Dir = dir
+	out, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(string(out)), nil
+}
+
+func SearchVersionsLocal(cmdCtx *cmdcontext.CmdCtx, program string) error {
+	var err error
+	cliOpts, err := configure.GetCliOpts(cmdCtx.Cli.ConfigPath)
+	if err != nil {
+		return err
+	}
+	if cliOpts.Repo == nil {
+		cliOpts.Repo = &config.RepoOpts{Install: "", Rocks: ""}
+	}
+	localDir := cliOpts.Repo.Install
+	if localDir == "" {
+		localDir = cmdCtx.Cli.ConfigPath + "/distfiles"
```

Comment:
As I know configpath if a file path, but not a dir. How does this work?
Anyway, use filepath Join.

Reply:
changed